src/version.h

```diff
@@ -0,0 +1,11 @@
+#ifndef __VERSION_H_
+#define __VERSION_H_
+
+#define _STR(x)                 #x
+#define STR(x)                  _STR(x)
+#define VTX_VERSION_MAJOR       0 // increment when a major release is made (big new feature, etc)
+#define VTX_VERSION_MINOR       1 // increment when a minor release is made (small new feature, change etc)
+#define VTX_VERSION_PATCH_LEVEL 0 // increment when a bug is fixed
```

Comment:
nitpick: i would move those the the beginning of the files as the are the only thing user adjustable here